addons/xpaas-templates/xpaas-templates.addon

```diff
@@ -0,0 +1,102 @@
+oc create -f xpaas-templates/jws30-tomcat7-postgresql-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-basic-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-https-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mongodb-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mongodb-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-postgresql-s2i.json -n openshift
+
+oc create -f xpaas-templates/karaf2-camel-amq-template.json -n openshift
+oc create -f xpaas-templates/karaf2-camel-log-template.json -n openshift
+oc create -f xpaas-templates/karaf2-camel-rest-sql-template.json -n openshift
+oc create -f xpaas-templates/karaf2-cxf-rest-template.json -n openshift
+
+oc create -f xpaas-templates/openjdk18-web-basic-s2i.json -n openshift
+
+oc create -f xpaas-templates/processserver63-amq-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-amq-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-amq-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-amq-postgresql-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-basic-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-postgresql-s2i.json -n openshift
+
+oc create -f xpaas-templates/spring-boot-camel-amq-template.json -n openshift
+oc create -f xpaas-templates/spring-boot-camel-config-template.json -n openshift
+oc create -f xpaas-templates/spring-boot-camel-drools-template.json -n openshift
+oc create -f xpaas-templates/spring-boot-camel-infinispan-template.json -n openshift
+oc create -f xpaas-templates/spring-boot-camel-rest-sql-template.json -n openshift
+oc create -f xpaas-templates/spring-boot-camel-teiid-template.json -n openshift
+oc create -f xpaas-templates/spring-boot-camel-template.json -n openshift
+oc create -f xpaas-templates/spring-boot-camel-xml-template.json -n openshift
+oc create -f xpaas-templates/spring-boot-cxf-jaxrs-template.json -n openshift
+oc create -f xpaas-templates/spring-boot-cxf-jaxws-template.json -n openshift
+
+oc create -f xpaas-templates/sso70-https.json -n openshift
+oc create -f xpaas-templates/sso70-mysql-persistent.json -n openshift
+oc create -f xpaas-templates/sso70-mysql.json -n openshift
+oc create -f xpaas-templates/sso70-postgresql-persistent.json -n openshift
+oc create -f xpaas-templates/sso70-postgresql.json -n openshift
```

Comment:
Let's also add an echo at the end giving some sort of feedback what just happened.
Something like:

    echo Ansible OpenShift image streams and templates installed.
    echo See https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_examples/files/examples/v3.6.